source/Camera/Maintenance/VerifyCameraTrueness/VerifyCameraTrueness.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+This example shows how to verify the local dimension trueness of a camera. If the
+dimension trueness is much worse than expected, the camera may have been damaged
+by shock in shipping or handling. If so, look at the CorrectCameraTrueness sample.
```

Comment:
in case you change the name of the samples, don't forget this one.

Reply:
Yeah I've done that mistake too many times before ;)